tests/Sdk/Unit/Support/HasTraceProvider.php

```diff
@@ -4,8 +4,8 @@
 
 namespace OpenTelemetry\Tests\Sdk\Unit\Support;
 
-use OpenTelemetry\Sdk\Trace\TracerProvider;
 use OpenTelemetry\Sdk\Trace\Tracer;
+use OpenTelemetry\Sdk\Trace\TracerProvider;
```

Comment:
I'm assuming this was just an editor preference swapping these two locations?

Reply:
Actually I think cs-fixer is to blame.